test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -7728,6 +7728,34 @@ def forward(self, input_ids):
         self.run_test(M(), (x,), input_names=['input_ids'],
                       dynamic_axes={'input_ids': {0: 'batch', 1: 'sequence'}})
 
+    @skipIfUnsupportedMinOpsetVersion(9)
+    def test_hann_window(self):
+        class HannWindowModule_Periodic(torch.nn.Module):
+            def __init__(self):
```

Comment:
nit: init function is not needed since self.window_length is overwritten in forward.

Reply:
> nit: init function is not needed since self.window_length is overwritten in forward.

It is necessary for ScriptModule created via torch.jit.script(), so we need to keep them.